ehttp/src/native.rs

```diff
@@ -62,3 +64,16 @@ pub(crate) fn fetch(request: Request, on_done: Box<dyn FnOnce(crate::Result<Resp
         .spawn(move || on_done(fetch_blocking(&request)))
         .expect("Failed to spawn ehttp thread");
 }
+
+pub(crate) async fn fetch_async(request: Request) -> crate::Result<Response> {
```

Comment:
that pub modifier will be breaking semver, we might want to keep it pub, but add a comment pointing to the cross-platform fetch_async?